appinventor/blocklyeditor/src/msg/en/_messages.js

```diff
@@ -384,18 +384,28 @@ Blockly.Msg.en.switch_language_to_english = {
     Blockly.Msg.LANG_MATH_ARITHMETIC_HELPURL_MULTIPLY = '/reference/blocks/math.html#multiply';
     Blockly.Msg.LANG_MATH_ARITHMETIC_HELPURL_DIVIDE = '/reference/blocks/math.html#divide';
     Blockly.Msg.LANG_MATH_ARITHMETIC_HELPURL_POWER = '/reference/blocks/math.html#exponent';
+    Blockly.Msg.LANG_MATH_BITWISE_HELPURL_AND = '/reference/blocks/math.html#bitwise-and';
```

Comment:
In the HTML the names use `_` but here they use `-`. This means that when one selects Help from the context menu, the browser doesn't jump to the right location. Either make both be `_` or `-`, but don't mix in the URL.

Reply:
Whoops. Fixed.